packages/react-percy-webpack/src/compile/createCompiler.js

```diff
@@ -29,16 +29,20 @@ export default function createCompiler(percyConfig, webpackConfig) {
           ],
         };
 
-  return webpack(
-    merge(webpackConfig, {
-      output: {
-        chunkFilename: '[name].chunk.js',
-        filename: '[name].js',
-        path: path.join(percyConfig.rootDir, 'static'),
-        publicPath: '/',
-      },
-      module,
-      plugins: [new MemoryOutputPlugin('/static/')],
-    }),
+  const mergedWebpackConfig = merge(webpackConfig, {
+    output: {
+      chunkFilename: '[name].chunk.js',
+      filename: '[name].js',
+      path: path.join(percyConfig.rootDir, 'static'),
+      publicPath: '/',
+    },
+    module,
+    plugins: [new MemoryOutputPlugin('/static/')],
+  });
+
+  mergedWebpackConfig.plugins = mergedWebpackConfig.plugins.filter(
+    plugin => !(plugin instanceof webpack.optimize.CommonsChunkPlugin),
```

Comment:
As we experiment with more projects, we'll have to see whether it makes sense to continue using people's existing webpack configs. The alternative is to do something more like storybook, where we have our own tightly controlled webpack config but expose a way for people to add additional loaders and plugins.

My hope is to continue using people's existing webpack configs so it's less work to get started and can work easily out of the box, but if we find lots of cases like this where we're needing to strip out rules or plugins from their configs, we may need to rethink that.